string robot_id
---
bool success